src/shogun/mathematics/linalg/ratapprox/logdet/opfunc/LogRationalApproximationIndividual.cpp

```diff
@@ -7,36 +7,34 @@
 #include <shogun/lib/config.h>
 
 #include <shogun/base/Parameter.h>
-#include <shogun/lib/SGVector.h>
 #include <shogun/lib/SGMatrix.h>
-#include <shogun/mathematics/linalg/linsolver/LinearSolver.h>
+#include <shogun/lib/SGVector.h>
+#include <shogun/mathematics/eigen3.h>
 #include <shogun/mathematics/linalg/linop/DenseMatrixOperator.h>
 #include <shogun/mathematics/linalg/linop/SparseMatrixOperator.h>
+#include <shogun/mathematics/linalg/linsolver/LinearSolver.h>
 #include <shogun/mathematics/linalg/ratapprox/logdet/opfunc/LogRationalApproximationIndividual.h>
-#include <shogun/mathematics/linalg/ratapprox/logdet/computation/job/RationalApproximationIndividualJob.h>
-#include <shogun/mathematics/linalg/ratapprox/logdet/computation/aggregator/IndividualJobResultAggregator.h>
-#include <shogun/lib/computation/engine/IndependentComputationEngine.h>
 #include <typeinfo>
 
+using namespace Eigen;
+
 namespace shogun
 {
 
-CLogRationalApproximationIndividual::CLogRationalApproximationIndividual()
-	: CRationalApproximation(NULL, NULL, NULL, 0, OF_LOG)
-{
-	init();
+	CLogRationalApproximationIndividual::CLogRationalApproximationIndividual()
+	    : CRationalApproximation(NULL, NULL, 0, OF_LOG)
+	{
+		init();
 
-	SG_GCDEBUG("%s created (%p)\n", this->get_name(), this)
+		SG_GCDEBUG("%s created (%p)\n", this->get_name(), this)
```

Comment:
Delete this, not needed

Reply:
I deleted it from all opfunc classes because it appears to be deprecated. is it okay ?